docs/crocks/Maybe.md

```diff
@@ -0,0 +1,97 @@
+# Maybe
+
+`Maybe c a`
```

Comment:
While technically the siggy for `Maybe` is: `Maybe () a`,  but because the first parameter is fixed to `Unit`, we just never show it. So typically the siggy for `Maybe` is `Maybe a`.